Dizzy/Entities/Images.swift

```diff
@@ -85,4 +85,12 @@ struct Images {
     static func placeBackgroundIcon() -> UIImage {
         return UIImage(named: "placeBackgroundIcon")!
     }
+    
+    static func showIcon() -> UIImage {
```

Comment:
this name is not good,  if i reed this line, i dont know which icon this is...
you need to call it "showXicon" or something like that